### ISSUE TITLE
Regression test failures

### ISSUE THREAD

**Issue by karllark** [BEAST-Fitting/beast#598]
We have updated things in the BEAST that are causing some of the regression tests to fail.  The cached files used by these tests need updating.  Most of the standard cached files (i.e., generated by running the phat_small example) have already been updated.   The remaining regression tests that need updated files or revising so that they are independent of the cached files are:

- [ ] test_star_type_probability_all_params
- [ ] test_compare_spec_type_inFOV
- [ ] test_ast_pick_models
- [ ] test_read_lnp_data
- [ ] test_read_noise_data

**Comment by lea-hagen** [comment]
I'm pretty sure the underlying reason for the failures of these is the number of PDF bins used in the fitting.  Going from 50 to 100 bins changes the 1D/2D PDFs (obviously) as well as the stats.  I keep getting mixed up with how many bins the tests should be using, so it would help to clarify that.

**Comment by karllark** [comment]
I don't think it is the number of bins, but that the SED file has changed due to removing the LogL < -9. points from the isochrones.

**Comment by lea-hagen** [comment]
Oh yeah, that too!

**Comment by karllark** [comment]
These tests are marked to be skipped in PR #596.  When each test is fixed, the skip notation should be removed.   Skipping the tests allows for the coverage report to be generated which is useful.
